sdscli/adapters/hysds/update.py

```diff
@@ -40,12 +40,9 @@
 def update_mozart(conf, ndeps=False, config_only=False, comp='mozart'):
     """"Update mozart component."""
 
-    # number of progress bar updates
-    num_updates = 24 if config_only else 37
-
-    # progress bar
-    with tqdm(total=num_updates) as bar:
+    num_updates = 24 if config_only else 37  # number of progress bar updates
```

Comment:
```suggestion
    num_updates = 27 if config_only else 41  # number of progress bar updates
```

Reply:
@DustinKLo: Let's update/correct the total number of `bar.update()` calls we expect depending on whether or not `config_only` was set with the `-c` option. I count 41 `bar.update()` calls within the `with` context below if `config_only` is False and 27 otherwise. You count that too?